.github/workflows/ci.yml

```diff
@@ -25,37 +25,15 @@ jobs:
         with:
           path: |
             node_modules
-          key: ${{ runner.OS }}-${{ matrix.node }}-yarn-cache-${{ hashFiles('**/yarn.lock') }}
+          key:
+            ${{ runner.OS }}-${{ matrix.node }}-yarn-cache-${{
+            hashFiles('**/yarn.lock') }}
           restore-keys: ${{ runner.OS }}-${{ matrix.node }}-yarn-cache-
 
       - run: yarn --no-progress --non-interactive --frozen-lockfile
 
-      - run: yarn build
-      - run: yarn lint
+      - name: 'TypeCheck, Lint and Build'
+        # NextJS typechecks and lints before the build
+        run: yarn build
```

Comment:
```suggestion
      - run: yarn build
```

Reply:
Will this work @LogvinovLeon 
? The object starts with "name"